### DISCUSSION TITLE
Ability to close currently selected window with F4 in Window switcher

### DISCUSSION THREAD

**Discussion by sanderjoon** [valinet/ExplorerPatcher#1463] [Ideas]
Not all keyboards have the Delete key.

🙏

**Comment by sanderjoon** [comment]
Generally ability to change shortcuts would be quite helpful. Alt+ˇ letter is not so common either.

**Comment by Cyan256** [comment]
You should consider using the PowerToys in: https://github.com/microsoft/PowerToys/releases/tag/v0.70.0
Choose the options on the Keyboard Shortcut
